src/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests.csproj

```diff
@@ -19,6 +19,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.169" />
+    <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```

Comment:
Why do we need this now?

Reply:
There were sonar cloud warning so i updated some of the packages 
<img width="544" alt="image" src="https://github.com/user-attachments/assets/41855a5c-a3f1-47f5-a958-88f6a41862e2">